hypothesis-python/src/hypothesis/extra/ghostwriter.py

```diff
@@ -475,7 +475,7 @@ def _is_probably_ufunc(obj):
     (r"dump(.+)", "load{}"),
     (r"to(.+)", "from{}"),
     # Known stem, maybe matching prefixes, maybe matching postfixes.
-    (r"(.*)encode(.*)", "{}decode{}"),
+    (r"(.*)en(.+)", "{}de{}"),
```

Comment:
I wondered about false positives, so I wrote quick script to scan `/usr/share/dict/words` for matching pairs that aren't actually en/de pairs, and it found these, for the record:

* ensign, design
* entail, detail
* enter, deter
* envelop, develop
* envious, devious

Reply:
I should have put this in the PR description... There are indeed a few undesired matches, but I think they're sufficiently rare (and low-impact) that it's a good tradeoff.